client/state/data-layer/wpcom/signup/verticals/test/index.js

```diff
@@ -27,6 +27,7 @@ describe( 'data-layer/wpcom/signup/verticals', () => {
 						site_type: mockAction.siteTypeId,
 						limit: mockAction.limit,
 						include_preview: true,
+						synonyms: true,
```

Comment:
Following the suggested update to D32511-code

```suggestion
				allow_synonyms: true,
```